lapis/job.py

```diff
@@ -28,8 +30,10 @@ def __init__(
         self,
         resources: dict,
         used_resources: dict,
+        cpu_efficiency: Optional[float] = None,
```

Comment:
What is your idea to specifically introduce the parameter `cpu_efficiency`? As it is currently defined `cpu_efficiency` equals `used_resources["cores"]`.

Reply:
Ah, yes. I forgot to remove this after reading your answer on used_resources["cores"]